src/App.tsx

```diff
@@ -1,4 +1,5 @@
 import { useEffect, useState } from 'react';
+require('./globals.css');
```

Comment:
You already have an import in https://github.com/welschoff/worth-watching-series/blob/login/src/main.tsx#L3